readme.md

```diff
@@ -38,40 +38,66 @@ The Autocomplete functionality of Notepad++ is not accessible by default. The au
 As you type, the document scrolls to show the line of text with the most likely phrase you are looking for. It also highlights the text that matched.
 The program also shows you how many matches have been detected. There are buttons to move to the next and previous match.
-As you type, NVDA will announce the line of text that notepad++ detected a search result in. NVDA also announces how many matches there are, but only if the number of matches has changed. 
+As you type, NVDA will announce the line of text that Notepad++ detected a search result in. NVDA also announces how many matches there are, but only if the number of matches has changed. 
 When you found the line of text you want, simply press escape, and that line of text will be at your cursor.
 To launch this dialog, select incremental search from the search menu, or press alt+control+i.
 
-### Reporting information about the  current line
+### Reporting Information about the Current Line
 
 Pressing nvda+shift+\ (back slash) at any time will report the following:
 
 * the line number
 * the column number I.E. how far into the line you are.
-* the selection size, (number of characters horizontally selected, followed by  the number of characters vertically selected, which would make a rectangle. This info is only reported if relevant.
+* the selection size, (number of characters horizontally selected, followed by the number of characters vertically selected, which would make a rectangle. This info is only reported if relevant.
 
-### Support for the previous/next find feature.
+### Support for the Previous/Next Find Feature
 
 By Default, if you press control+f you bring up the find dialog. 
 If you type text here and press enter, the text in the window is selected and the document is moved to the next search result. 
 In Notepad++ you can press f3 or shift+f3 to repeat the search in the forward or backward direction respectively. 
 NVDA will read both the current line, and the selection within the line which represents the found text.
 
-## Non-default Notepad++ keyboard shortcuts
+### Preview of MarkDown or Hypertext as Webpage
+
+Notepad++ does natively not support MarkDown (*.md) with e.g. language highlighting. 
+However, you can preview such content as browsable message if you press NVDA+H (Escape to close the message). 
+Pressing this combination twice will open it in your standard browser. 
+Some popular Markdown extensions such as PHP Extra or TOC are supported. 
+It works also with (single-paged) Html. 
+
+To try it out, Copy the following block, paste it into a new Notepad++ document and press NVDA+H:
+
+<br>
+
+    ---
+    ## Where it began...  
+    > A long time ago,  
+    > in a foreign country.  
+    ## And where it went next  
+    1. First stage  
+    2. Second stage  
+    ## Eventually it became  
+    * unordered  
+    * but still  
+    * a list  
+
+<br>
+
+# NonDdefault Notepad++ Keyboard Shortcuts
```

Comment:
NIT: Wrong heading level, ddefault should be default.

Reply:
Done
change to heading three.
NonDdefault -> Non-Default